src/styles/blocks/blocks.scss

```diff
@@ -0,0 +1,21 @@
+.container {
+  display: grid;
+
+  &__block {
+    display: flex;
+    justify-content: center;
+    align-items: center;
+    color: white;
+    min-width: $blockHeight;
+    height: $blockHeight;
+    font-size: 100px;
+    flex-grow: 1;
+    flex-basis: calc(100% / 3);
```

Comment:
Pay attention to `css` properties grouping - https://codeguide.academy/html-css.html#css-order

Reply:
done